pkg/apis/hive/v1/hiveconfig_types.go

```diff
@@ -125,7 +125,9 @@ type VeleroBackupConfig struct {
 // FailedProvisionConfig contains settings to control behavior undertaken by Hive when an installation attempt fails.
 type FailedProvisionConfig struct {
 
-	// SkipGatherLogs disables functionality that attempts to gather full logs from the cluster if an installation
+	// TODO: Figure out how to mark SkipGatherLogs as deprecated (more than just a comment)
```

Comment:
I can't seem to find a way to mark SkipGatherLogs as deprecated, other than what I did below with the comment.

I checked kubebuilder markers and didn't find anything related to deprecation. Did I miss one?

Does anyone know how to mark these as deprecated?

Reply:
Looks like the deprecated attribute was introduced in upstream in version 1.19 (ocp 4.6)
https://github.com/kubernetes/kubernetes/pull/92329
I doubt that there is tooling support for it yet

Reply:
ok, thanks!

I guess the comment will have to be good enough for now. :)